static/usr/lib/core/fix-hctosys

```diff
@@ -17,6 +17,9 @@
 #exec 2>&1
 #printf "%s\n" "Starting $(date)"
 
+DEV_PATH="$1"
```

Comment:
Maybe we do not need to pass as parameter.

Though it would be nice to check whether the device comes from a module. Because in that case we do not need to fix anything.

That said, we can also use an `IMPORT{program}` with the following script to do some filtering the udev rule instead.

```
#!/bin/sh
set -eu

device=/sys/${DEVPATH}

while true; do
    if [ -d "${device}/driver" ]; then
        if [ -L "${device}/driver/module" ]; then
            echo FROM_MODULE="$(basename "$(realpath "${device}/driver/module")")"
        fi
        break
    elif [ -d "${device}/device" ]; then
        device="${device}/device"
    else
        break
    fi
done
```

Reply:
Do not worry about this comment. This is the least urgent.

Reply:
Moved away from RUN to SYSTEMD_WANTS based approach using same service template as the generator. To my understanding this limits ability to "pass arguments" to the service in a simple way.